io/src/ply_io.cpp

```diff
@@ -1209,14 +1209,36 @@ pcl::PLYWriter::writeBinary (const std::string &file_name,
   }
 
   std::ofstream fs;
-  fs.open (file_name.c_str ());      // Open file
+  fs.open (file_name.c_str (),
+           std::ios::out | std::ios::binary |
+               std::ios::trunc); // Open file in binary mode and erase current contents
+                                 // if any
   if (!fs)
   {
-    PCL_ERROR ("[pcl::PLYWriter::writeBinary] Error during opening (%s)!\n", file_name.c_str ());
+    PCL_ERROR ("[pcl::PLYWriter::writeBinary] Error during opening (%s)!\n",
+               file_name.c_str ());
     return (-1);
   }
 
-  unsigned int nr_points  = cloud.width * cloud.height;
+  if (!(writeBinary (fs, cloud, origin, orientation, use_camera) == 0))
+  {
+    fs.close();
+    return -1;
+  }
+  fs.close();
+  return 0;
+}
+
+int
+pcl::PLYWriter::writeBinary (std::ostream& fs,
```

Comment:
nitpick - since its not necessarily a file stream, maybe another abbreviation would be appropriate? os?

Reply:
Absolutely. I will make the suggested change